CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for crate
 Unreleased
 ==========
 
+- Stopped decoding the ``timeout`` parameter strictly as scalar to also
```

Comment:
Please use a user facing message here, so what has changed for the user, not the internal implementation details.

Reply:
Thanks. I've amended both the changelog item, and the commit message according to your suggestions.